app/views/payments/EstimatedILPP2View.scala.html

```diff
@@ -0,0 +1,119 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.viewModels.EstimatedLPP2ViewModel
+@import views.templates.formatters.dates.DisplayDateRangeHelper._
+@import views.html.templates.formatters.money.DisplayMoney
+
+@this(mainTemplate: MainTemplate,
+      displayMoney: DisplayMoney,
+      govukBreadcrumbs: GovukBreadcrumbs,
+      govukSummaryList: GovukSummaryList,
+      govukBackLink: GovukBackLink)
+
+@(model: EstimatedLPP2ViewModel, serviceInfoContent: Html)(
+  implicit request: Request[_], messages: Messages, appConfig: config.AppConfig, user: User)
+
+@breadcrumbs = {
+  @govukBreadcrumbs(Breadcrumbs(
+    items = Seq(
+      BreadcrumbsItem(
+        content = Text(messages("breadcrumbs.bta")),
+        href = Some(appConfig.btaHomeUrl)
+      ),
+      BreadcrumbsItem(
+        content = Text(messages("vatDetails.title")),
+        href = Some(controllers.routes.VatDetailsController.details.url)
+      ),
+      BreadcrumbsItem(
+        content = Text(messages("openPayments.title")),
+        href = Some(testOnly.controllers.routes.WhatYouOweController.show.url)
+      )
+    )
+  ))
+}
+
+@backLink = {
+  @govukBackLink(BackLink(
+    content = Text(messages("base.back")),
+    href = testOnly.controllers.routes.WhatYouOweController.show.url
+  ))
+}
+
+@mainTemplate(
+  title = model.title,
+  appConfig = appConfig,
+  serviceInfoContent = serviceInfoContent,
+  user = Some(user),
+  navLinkContent = if(user.isAgent) Some(backLink) else Some(breadcrumbs)
+) {
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-xl" id="estimated-lpp2-heading">
+          <span class="govuk-caption-xl">
+            @displayDateRange(model.periodFrom, model.periodTo, alwaysUseYear = true)
+          </span>
+            @model.title
+      </h1>
+
+      <p class="govuk-body">@messages("estimatedLPP2.explanation", model.day)</p>
+      <p class="govuk-body">@messages("estimatedLPP2.interestExplanation")</p>
+
+      <p class="govuk-body">
+        @messages("estimatedInterest.calculation")
+        <br>
+        @messages("estimatedLPP2.calculationExplanation", model.penaltyRate)
+      </p>
+
+      @govukSummaryList(SummaryList(
+        rows = Seq(
+          SummaryListRow(
+            key = Key(content = Text(messages("estimatedLPP2.penaltyAmount"))),
+            value = Value(content = HtmlContent(displayMoney(model.penaltyAmount)))
+          ),
+          SummaryListRow(
+            key = Key(content = Text(messages("estimatedLPP2.amountReceived"))),
+            value = Value(content = HtmlContent(displayMoney(0.00)))
```

Comment:
Can you change the value to simply `0`. Our money helper will trim trailing 0s in the decmal places anyway. This is different from the prototype screenshot but is in line with GOV.UK style guidance: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#money